recipes/quast/meta.yaml

```diff
@@ -1,44 +1,51 @@
+{% set name = "quast" %}
+{% set version = "4.5" %}
+{% set hash = "873ed8cd69e2d92fb87d75a37ecb75ea" %}
```

Comment:
Yes, as I said, you can keep it like that. Indeed, the hash changes, but it occurs only at one place in the file so one could also simply modify it there (and have less clutter in the syntax).